code/modules/projectiles/mounted.dm

```diff
@@ -286,18 +260,19 @@
 		stack_trace("[src] has its view_tile offset set higher than 7, please don't.")
 
 	user.client.change_view(WORLD_VIEW)
+	var/view_offset = view_tile_offset * 32
```

Comment:
why 32?
and use a define if it will always stay 32

Reply:
32 is the pixel size of tiles, i stole the 32 from tl102 code and from what i can tell it keeps the screen the same view. I could always see what i could steal from scope code

Reply:
it should use the proper define then